projects/rocprofiler-systems/source/lib/core/trace_cache/cache_manager.cpp

```diff
@@ -189,6 +188,32 @@ cache_manager::post_process_bulk()
                 thread.join();
             }
         }
+
+        ROCPROFSYS_PRINT("Removing temporary files...\n");
+
+        auto remove_if_exists = [](const std::string& fname) {
```

Comment:
Add a check for an empty string?

```suggestion
        auto remove_if_exists = [](const std::string& fname) {
            if(fname.empty()) return;

```